engine/src/test/scala/org/scalarules/utils/GlossaryTest.scala

```diff
@@ -3,6 +3,7 @@ package org.scalarules.utils
 import org.scalarules.engine.SingularFact
 import org.scalatest.{FlatSpec, Matchers}
 
+@deprecated(message = "Glossary is deprecated in favor of MacroGlossary", since = "0.3.0")
```

Comment:
If you have a better way of shutting the compiler up about using deprecated API... ;) 

I thought it fitting in a way, to also expressly indicate that this test is testing something that's deprecated. After all, it too will be deleted once the code under test is removed.